src/main/Config.h

```diff
@@ -492,9 +492,6 @@ class Config : public std::enable_shared_from_this<Config>
     // fees, and scp history in the database
     bool MODE_STORES_HISTORY_MISC;
 
-    // A config parameter that stores ledger headers in the database
-    bool MODE_STORES_HISTORY_LEDGERHEADERS;
```

Comment:
We need to check with platform to make sure they don't do anything with this flag, otherwise we need to wait for the next major release.